### PR TITLE
add new schema types for integers and empty schemas

### PR DESCRIPTION
**Pull request by zachharris1**
# Description of change
This pr adds schemas types for integers and undefined schema types.

This pr has not been thoroughly tested.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by dmosorast — APPROVED** [review]
Approving because this is safe to merge if we really want to, if we cannot get information about what the data type actually should be in these situations (read: where `schema_type == ''`).

The effect of this would be if someone selects a field that has this schema and receives a non-null value, target-stitch would blow up visibly, and would provide an actual use case to inspect. I would expect all other targets to behave the same (if they validate json schema).

On the other hand, if we really need to merge this, we can, because it'd get users unstuck who have schemas like this, but don't want to select that particular field, less-than-ideal, but a safe stop-gap.

File: tap_pendo/discover.py [hunk 1]
```diff
@@ -56,6 +56,10 @@ def get_schema_property_type(schema_type):
         return {"type": ["null", "string"], "format": "date-time"}
     elif schema_type == 'boolean':
         return {"type": ["null", "boolean"]}
+    elif schema_type == 'integer':
+        return {"type": ["null", "integer"]}
+    elif schema_type == '':
+        return {}
```
Comment (zachharris1) [inline tap_pendo/discover.py:62]: Maybe personal preference, but I like `dict()` over `{}` to remove ambiguity between whether its a set or dict.

**Review by dmosorast — APPROVED** [review]
Seems good. 👍

File: tap_pendo/discover.py (hunk 1 above)
Comment (zachharris1) [reply tap_pendo/discover.py:62]: I approve with or without this change, unfortunately I can't approve my own pr ☹️

**Pull request merged**